### DISCUSSION TITLE
While i'm migrating teleport cluster from one server to another i'm getting this error need solutions of this one.

### DISCUSSION THREAD

**Discussion by tejasri354970** [gravitational/teleport#30122] [Q&A]
2023-08-07T07:51:23Z INFO             Starting Teleport v13.2.2 with a config file located at "/etc/teleport.yaml" common/teleport.go:556
2023-08-07T07:51:23Z ERRO [SQLITE]    "Failing schema step: CREATE TABLE IF NOT EXISTS kv (\n           key TEXT NOT NULL PRIMARY KEY,\n           modified INTEGER NOT NULL,\n           expires DATETIME,\n           value BLOB);\n        CREATE INDEX IF NOT EXISTS kv_expires ON kv (expires);, unable to open database file: permission denied." lite/lite.go:331
ERROR: error creating schema: file:/var/lib/teleport/proc/sqlite.db?_busy_timeout=10000&_sync=FULL&_txlock=immediate
initialization failed
unable to open database file: permission denied

**Comment by webvictim** [comment]
This looks like a simple permissions issue.

- What user is Teleport running as? (run `ps -o user $(pidof teleport)`)
- What is the ownership of `/var/lib/teleport/backend/sqlite.db` (run `sudo ls -l /var/lib/teleport/backend/sqlite.db`)

**Reply by tejasri354970** [reply]
yes did it, but teleport won't start.

**Reply by webvictim** [reply]
What error do you get? Can you share the output of those commands?

**Reply by tejasri354970** [reply]
Hi, when I'm trying to check the teleport service it's working.
But I cant access web UI .
1. Teleport user 
USER
root
2. Ownership was 
root
when it try to access the Web UI its shows 502 bad gateway.
the error im getting now
Oct 20 16:07:04 teleport[958]:         runtime/asm_amd64.s:1598 runtime.goexit
Oct 20 16:07:04  teleport[958]: User Message: acme/autocert: missing server name] alpnproxy/proxy.go:337
Oct 20 16:07:05  teleport[958]: 2023-10-20T16:07:05Z WARN [ALPN:PROX] Failed to handle client connection. error:[
Oct 20 16:07:05 ip teleport[958]: ERROR REPORT:
Oct 20 16:07:05  teleport[958]: Original Error: *errors.errorString acme/autocert: missing server name
Oct 20 16:07:05  teleport[958]: Stack Trace:
Oct 20 16:07:05 teleport[958]:         github.com/gravitational/teleport/lib/srv/alpnproxy/proxy.go:392 github.com/gravitational/tele>
Oct 20 16:07:05  teleport[958]:         github.com/gravitational/teleport/lib/srv/alpnproxy/proxy.go:326 github.com/gravitational/tele>
Oct 20 16:07:05  teleport[958]:         runtime/asm_amd64.s:1598 runtime.goexit
Oct 20 16:07:05 teleport[958]: User Message: acme/autocert: missing server name] alpnproxy/proxy.go:337

**Reply by webvictim** [reply]
`502 Bad Gateway` implies you have a load balancer or reverse proxy in front of your cluster - how is that configured?

Can you also please share the Teleport config file you're using (`/etc/teleport.yaml`)?

**Reply by tejasri354970** [reply]
I'm using ALB and the service is up but my nodes are not connected.
I got this error now. How to resolve this issue my users need to get access to all the nodes as usual.

Error 
disconnected
Teleport proxy failed to connect to "node" agent "@local-node" over reverse tunnel:

  no tunnel connection found: no node reverse tunnel for c8f3074e-c793-4a34-85cc-2600eacba96c.teleport.sudheer.com found

This usually means that the agent is offline or has disconnected. Check the
agent logs and, if the issue persists, try restarting it or re-registering it
with the cluster.

**Reply by webvictim** [reply]
Can you show how the ALB is configured? Specifically, is it using an HTTPS connection to the Teleport backend (as it needs to be)?

**Reply by tejasri354970** [reply]
yes, Thanks for your help it's connected.
But I cant log into my existing nodes. its showing disconnected.

**Reply by tejasri354970** [reply]
Hello @webvictim 
I've another query i want to deploy teleport behind the open VPN. whenever my users access to web UI they need to connect to the VPN and after that, they need to access the resources.
I tried the chat GPT suggestions but it is showing this error 

The error message you're encountering indicates that Teleport doesn't recognize the openvpn_service section in your configuration because it's not a built-in part of Teleport's configuration.
could you please help me out with this one?
